packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -1111,6 +1111,26 @@ typedef SelectableDayPredicate = bool Function(DateTime day);
 /// The [context] argument is passed to [showDialog], the documentation for
 /// which discusses how it is used.
 ///
+/// The [builder] parameter can be used to wrap the dialog widget to change
```

Comment:
to change to add? Probably just "to add"?

Reply:
(here and int he other file)

Reply:
Yes, thanks for catching that.